myst_nb/parser.py

```diff
@@ -37,6 +41,25 @@ def parse(self, inputstring, document):
         # de-serialize the notebook
         ntbk = nbf.reads(inputstring, nbf.NO_CONVERT)
 
+        self.reporter = document.reporter
+        self.config = self.default_config.copy()
+
+        try:
```

Comment:
I'm not sure if there is a good reason for all this being in a try/except block

Reply:
the block was already there, with some variables, I just added mine. Probably, I should move mine away from it.

Reply:
yeh maybe @choldgraf the less in try/except the better

Reply:
definitely feel free to clobber try/except in order to do a more rigorous check...I think a lot of this was just getting to MVP's for iteration purposes 👍

Reply:
thinking back on it, I think that this was actually because of testing - maybe it's changed now, but there used to be tests that were run that didn't initialize the full Sphinx app, so this was probably a hack around that. Maybe this is no longer an issue?